Library/OcCpuLib/OcCpuLib.c

```diff
@@ -853,17 +853,13 @@ ScanAmdProcessor (
   UINT64 CoreDivisorID;
   UINT64 Divisor;
   //
-  // Faking an Intel Core i5 Processor.
-  // This value is purely cosmetic, but it makes sense to fake something
-  // that is somewhat representative of the kind of Processor that's actually
-  // in the system
+  // Faking as unknown processor. This value is purely cosmetic.
   //
-  Cpu->AppleProcessorType = AppleProcessorTypeCorei5Type5;
+  Cpu->AppleProcessorType = AppleProcessorMajorUnknown;
   //
   // get TSC Frequency calculated in OcTimerLib
   //
   Cpu->CPUFrequencyFromTSC = GetPerformanceCounterProperties (NULL, NULL);
-  Cpu->CPUFrequency = Cpu->CPUFrequencyFromTSC;
   //
```

Comment:
I am not positive AMD will anyhow benefit from rounding below. What is the intent of this change?